lib/TBDGen/TBDGen.cpp

```diff
@@ -425,6 +425,43 @@ static bool protocolDescriptorHasRequirements(ProtocolDecl *proto) {
   return false;
 }
 
+#ifndef NDEBUG
+static bool isValidProtocolMember(Decl *D) {
```

Comment:
Nitpick: maybe `isValidProtocolMemberForTBDGen`? Because MissingMemberDecl can happen, just not in a file you're currently compiling. (Also, `const`.)